content/en/docs/contributing/release-process.md

```diff
@@ -258,33 +260,35 @@ The process for cutting a minor release is as follows:
       Cloud Build job link that `cmrel` displayed in "View logs at". For
       example, the message would look like:
 
-        _Follow the cmrel stage build: <https://console.cloud.google.com/cloud-build/builds/7641734d-fc3c-42e7-9e4c-85bfc4d1d547?project=1021342095237>_
+        {{% pageinfo %}}
+Follow the `cmrel stage` build: <https://console.cloud.google.com/cloud-build/builds/7641734d-fc3c-42e7-9e4c-85bfc4d1d547?project=1021342095237>
+        {{% /pageinfo %}}
 
 6. Run `cmrel publish`
+   
+   <div class="p-3 display-4">Background: Primary</div>
 
-   1. Find the "release name". The release name can be found in the output of
-      the previous command, `cmrel stage`. In the previous command output, look
-      for a line that looks like this:
+   1. Set the `RELEASE_NAME` variable in your shell by. The value for the
+      `RELEASE_NAME` variable is found in the output of the previous command,
+      `cmrel stage`. Look for the line that contains the `gs://` link:
```

Comment:
"by" in the first line needs removing:

> Set the `RELEASE_NAME` variable in your shell.

Reply:
Thanks again 😅 Fixed in 26443c8
I should proof-read everything I write at least once 😇